include/inviwo/core/processors/processorstate.h

```diff
@@ -35,7 +35,7 @@
 
 namespace inviwo {
 
-enum class CodeState { Broken, Experimental, Stable };
+enum class CodeState { Broken, Experimental, Stable, Deprecated };
```

Comment:
This should be in a different PR, have you also added icons and updated the GUI?

Reply:
Yes, the gui is update,  @martinfalk helped me with the icon. 
![image](https://user-images.githubusercontent.com/534670/43218470-c994d724-9044-11e8-9cb5-4ed8504fbed9.png)

Reply:
👍